### ISSUE TITLE
ID Exchange

### ISSUE THREAD

**Issue by FabianEbenhoch** [paulevsGitch/BetterEnd#22]
So I tried to use better end with my modpack. It works fine in the single player mode but when I start a Server it changes IDs of blocks as you can see in the picture. I already had the better end mod alone on the server so its not a problem causing of/with other mods. It changes the ID of seagrass or what its called (standard minecraft ID).
Thanks for helping me.
Manuki
![2020-11-22_02 53 00](https://user-images.githubusercontent.com/60819390/99891774-db770580-2c6d-11eb-8276-5d66606792ad.png)

**Comment by paulevsGitch** [comment]
BetterEnd doesn't change identifiers of any vanilla blocks. Here is a screenshot from server world with installed BetterEnd:

![2020-11-22_11 49 32](https://user-images.githubusercontent.com/17707380/99899292-29652b00-2cb9-11eb-95ed-6c5a46e069dd.png)

Actually all block IDs stored in block palettes, so this looks like semi-impossible:

![image](https://user-images.githubusercontent.com/17707380/99899305-4b5ead80-2cb9-11eb-8d2b-f1f82cff2b68.png)

Probably your world is corrupted, you can try to generate another one or manually fix your world with [WorldEdit](https://www.curseforge.com/minecraft/mc-mods/worldedit). If bug is still present than it is probably an API bug

**Comment by FabianEbenhoch** [comment]
The thing is that even if I start my Server and also my game only with "BetterEnd" it still have this bug. I also deleted the world more then once and let my server regenerate a new one. Yesterday I tried to set up my server from 0 but it didn't help. So the only thing that would make sense would be the API bug. My question now is how do I fix the API bug? Do I need to reinstall the Fabric API or something else?
Thanks for helping ^^
